src/wasm.rs

```diff
@@ -87,9 +99,7 @@ pub fn embed_claims(orig_bytecode: &[u8], claims: &Claims, kp: &KeyPair) -> Resu
 
     let digest = sha256_digest(cleanbytes.as_slice())?;
     let claims = (*claims).clone();
-    let metadata = claims.wascap_metadata.clone();
-    metadata.unwrap().module_hash = HEXUPPER.encode(digest.as_ref());
-
+    claims.wascap_metadata.unwrap().module_hash = HEXUPPER.encode(digest.as_ref());
```

Comment:
I think this is where my problem is, the metadata isn't being set & saved because I don't understand all this ownership.